app/controllers/users_controller.rb

```diff
@@ -7,7 +7,9 @@ def index
   end
 
   # GET /users/1 or /users/1.json
-  def show; end
+  def show
+    @user = User.find(params[:id])
+  end
 
   # GET /users/new
   def new
```

Comment:
As part of the project requiremens you were asked NOT to use `rails generate scaffold` command to generate the project/controllers since this project is to show you how to set up the controllers manually by yourself. However, I see that you have used the `rails generate scaffold`, hence a lot of code in the controller which are not needed at this milestone. Kindly consider removing all the boilerplate code generated by the `rails generate scaffold` and consider creating the controllers manually. 
The users_controller is suppose to have the #index and #show actions only for now.

Reply:
Aye Aye Captain